docs/DeveloperGuide.md

```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
```

Comment:
Do we think that patients and nurses are universally known terms? Maybe we can further contextualise the meaning of patients to those are require home visits?